nifti2/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 set(NIFTI_NIFTILIB2_NAME ${NIFTI_PACKAGE_PREFIX}nifti2)
 
 add_library(${NIFTI_NIFTILIB2_NAME} nifti2_io.c )
-target_link_libraries( ${NIFTI_NIFTILIB2_NAME} PUBLIC ${NIFTI_PACKAGE_PREFIX}niftiio)
+target_link_libraries( ${NIFTI_NIFTILIB2_NAME} PUBLIC ${NIFTI_PACKAGE_PREFIX}znz)
```

Comment:
@hjmjohnson out of interest what's the easiest way to work this out? The transitive dependency of niftiio prevents failure but is obviously inappropriate to use. Is there a tool that you use to work out the appropriate targets to link to or do you just start from nothing and start adding the correct targets that are required for linking?

Reply:
@leej3 What failure are you referencing?  I'm not sure what you are asking for.

Reply:
In this context, the question doesn't quite make sense. The goal of these changes was to remove niftiio as a dependency. 
Ignoring that context though. If nifti2 was dependent on znz and was failing to link, this could be resolved by linking to niftiio (since it exposes znz during the linking process anyway). The linking would work but the dependency would not really be correctly specified (or at least that is what I believe to be the case). My question was what was your process for determining what to link against. Do you just check the C code, start by linking nothing and work up from there, or is there a tool you use?